content/docs/main-concepts/api-clients.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: 'API Clients'
+section: 'Main Concepts'
+priority: 3
+hidden: false
+---
+
+import { Link } from 'gatsby'
+
+## What are API Clients?
+OrderCloud uses the term "API Clients" to identify various access points to your organization's data. These access points have properties that control what parties can use it, how they can gain access, and for how long that access remains valid.
+
+Think of each API client as a venue doorway; some are for staff or VIP, others are for the public, and they each have a lock or bouncer requiring the right type of key or ticket to get through. Once inside, a wristband is usually placed on the person's wrist - letting others know which door they came through.
+
+These locks are OAuth 2.0 Grant Types and they keys are the credentials needed to gain access to the data behind each API Client. The API Client used to gain access and other information about the user is encoded into an Access Token to be used on each subsequent request. This is the "wristband" that let's developers and the OrderCloud team know where and how a requestee gained access to the data.
```

Comment:
the consensus among the platform developers is that we should avoid analogies and more conversational writing in technical guides. Later on if we wanted to write something along the lines of "what the heck is an api client" blog post for new developers this analogy could be used in something like that.